### DISCUSSION TITLE
Loading sub-pakages  at startup

### DISCUSSION THREAD

**Discussion by burston** [iraf-community/iraf#289] [Q&A]
As a new github user I do not wish to waste anybodies time with non-orogramming details but as a lone user in regional Australia Imhave no other source of knowledge.
I have no need for hand holding but would very much kike to load sub-packqges at startup.
For example imred, ccdred, specred.
This II found was was easy to do in  2.16.1 but wuth 2.17.1 I have absolutely no idea how this may be achieved.
Any assistance would be gretly received.
If this is considered too trivial for GITHUB I appologise - a simple word along those lines would be appreciated so I could stop expecting a reply.
Hope this is clear.I am not a programmer.
Regards,
Patrick.

**Comment by olebole** [comment]
Can you explain how you did load sub-packages at startup in 2.16.1 and what happens if you do the same in 2.17?

**Reply by burston** [reply]
Thank you Ole - with further manipulations I managed to determine my ineptitude and have fixed the problem.
Regards,
Patrick.

**Comment by burston** [comment]
Also consider this closed.
Patrick.
